hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -996,77 +996,230 @@
     }
   ],
 
-  // ===== INTERRUPT CTRL =====================================================
-
-  // RV_PLIC has two searching algorithm internally to pick the most highest priority interrupt
-  // source. "sequential" is smaller but slower, "matrix" is larger but faster.
-  // Choose depends on the criteria. Currently it is set to "matrix" to meet FPGA timing @ 50MHz
-
-  // ===== ALERT HANDLER ======================================================
+  // ===== PINMUX & PINOUT ======================================================
+
+  // Pad declaration.
+  // The pad type maps to an enum that is defined inside the prim_pad_wrapper_pkg.
+  // Unmuxed pads that are not referenced as a DIO above are treated as "custom"
+  // dedicated pads that need to be connected by hand in the chip-level template.
+  pinout: {
+    pwr_domains: ['VCC', 'AVCC', 'VIOA', 'VIOB'],
+    type_default: 'BidirStd',
+    pwr_default: 'VCC',
+    muxed_default: 'True'
+    pads: [
```

Comment:
The motivation for the defaults was basically also to increase readability, since the whole table becomes sparser and has less entries.
However, it makes sense to make this explicit, so let me change that.